tests/test_unit_conversion.py

```diff
@@ -89,3 +87,18 @@ def test_conversion_factor_same_units(simple_model):
     assert simple_model.units.get_conversion_factor(to_unit=to_unit, from_unit=from_unit) == 1
     # expression to unit
     assert simple_model.units.get_conversion_factor(to_unit=to_unit, expression=expression) == 1
+
+
+def test_equivalent_units(model):
+    time_var = model.get_symbol_by_ontology_term(OXMETA, 'time')
+
+    assert model.units.equivalent_units(time_var, model.units.ureg.ms)
+    assert model.units.equivalent_units(time_var, model.units.ureg.ms)
+    assert model.units.equivalent_units(model.units.ureg.ms, time_var)
+
+    assert not model.units.equivalent_units(model.units.ureg.ms, model.units.ureg.second)
```

Comment:
Aren't the pure units cases tested below? There's definitely some test duplication here!